printer_mods/README.md

```diff
@@ -292,3 +292,4 @@ like so:
 | | [V0 Skirt set](./Zen3D/V0_Skirt)| SSR, PowerSwitch, Nameplate and color change printing  | :heavy_check_mark: |:x: |:x: |:x:|
 | | [V0 1515 T-nut](./Zen3D/V0_1515_T-nut)| M3 square and hex nuts for Voron0 1515 frame | :heavy_check_mark: |:x: |:x: |:x:|
 | zruncho | [V0 No-Drop Nuts](./zruncho/V0_No_Drop_Nuts) | Voron0 nut inserts which auto-align (LDO frames only) | :heavy_check_mark: | :x: | :x: | :x: |
+| xenon2008| [Wago & Klicky_Purge_Bucket](./xenon2008/Wago-Klicky_compatible_Purge_Bucket_Voron250) | Wago & Klicky compatible Purge Bucket | :x: | :x: | :heavy_check_mark: | :x: |
```

Comment:
Please move your line up so that it's in alphabetical order by username, looks like it should be between Xceled and Yeri.

Please remove underscores inside the square brackets, should be `[Wago & Klicky Purge Bucket]`

Please add an `:x:` in the Trident column.